tsfeatures/tsfeatures_r.py

```diff
@@ -1,155 +0,0 @@
-#!/usr/bin/env python
-# coding: utf-8
-
-from typing import List
```

Comment:
Can you please add this to a new notebook nbs/tsfeatures_r.ipynb?

Reply:
i added the notebook and extended the ci run to include R also added rpy2 in the dependencies list, @jmoralez maybe move it to the dev dependencies as it gives some troubles installing on windows